docs/plugin_development.rst

```diff
@@ -214,96 +266,122 @@ A minimal planning to organize ideas.
 .. note::
 
-   * `Anaconda Individual Edition`_ is a Python distribution for data science and machine learning to be used in a single machine.
+   * `Anaconda`_ is a Python distribution for data science and machine learning to be used in a single machine.
    * `Conda`_ is an Anaconda tool that manages virtual environments and their packages.
-   * Conda can work with *channels* that allow the use of packages that are not part of the official distribution. The most important channel is `conda-forge`_, where a more extensive and updated list of packages than those offered by Anaconda Individual Edition are maintained.
-   * Finally, `mamba`_, is an optimized implementation of conda's package management features, that resolves dependencies and installs packages much faster than conda.
+   * Conda can work with *channels* that allow the use of packages that are not part of the official distribution.
+     The most important channel is `conda-forge`_, where a more extensive and updated list of packages is maintained than those offered by Anaconda itself.
+   * Finally, `Mamba`_, is an optimized implementation of Conda's package management features, which resolves dependencies and installs packages much faster than Conda.
 
 .. _conda environment: https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html
-.. _installation guide: https://docs.spyder-ide.org/5/installation.html
-.. _Anaconda Individual Edition: https://www.anaconda.com/download
+.. _Anaconda: https://www.anaconda.com/download
 .. _Conda: https://docs.conda.io/en/latest/
 .. _conda-forge: https://conda-forge.org/
-.. _mamba: https://github.com/mamba-org/mamba
+.. _Mamba: https://github.com/mamba-org/mamba
 
 
+.. _tutorial-repo:
 
-===================
-Create a repository
-===================
+Creating a repository
+~~~~~~~~~~~~~~~~~~~~~~
 
-Now that we have our local virtual environment, it is good practice to manage our source code with a version control system, and the most widely used web service for this purpose is currently Github. Here you can find, for example, the Spyder and Python repositories.
+Now that we have our local virtual environment, it is good practice to manage our source code with a version control system, and the most widely used web service for this purpose is currently GitHub.
+Here you can find, for example, the Spyder and Python repositories.
 
 .. image:: images/workshop-3/pd_github_repo.png
-   :alt: Git and Github repository concepts.
+   :alt: Git and GitHub repository concepts
+
+To create a Git repository on GitHub, we need to follow these steps:
+
+#. Log in to your GitHub account.
+
+#. Click on the :guilabel:`New repository` option in the :guilabel:`+` menu at the top right next to your profile picture.
 
-To create a git repository on Github, we need to follow these steps:
+#. A dialog will appear where you can insert the repository name and some basic options, e.g.
+to initialize the repository with a README or license files.
```

Comment:
```suggestion
#. Type the repository name in the resulting dialog and select the options you want (e.g. whether to initialize the repository with a README or license files).
```

Fix syntax error, clarify this and rewrite it to be consistent in phrasing with the others